sqlobject/src/test/resources/org/jdbi/v3/sqlobject/statement/TestSqlScripts$Scriptacular/externalScript.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE temp (pk integer primary key);
```

Comment:
I guess the license checker wants too see a license boilerplate in the file header.